docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala

```diff
@@ -933,57 +443,6 @@ class VerifyYamlTest extends AnyFunSuite with Matchers {
     val actualYamlNoIndent = noIndentation(actualYaml)
     actualYamlNoIndent shouldBe expectedYaml
   }
-
-  test("should match the expected yaml when using schema with custom example") {
-    val expectedYaml = load("expected_schema_example.yml")
-
-    val expectedDateTime = ZonedDateTime.of(2021, 1, 1, 1, 1, 1, 1, UTC)
-    val expectedBook = Book("title", Genre("name", "desc"), 2021, Author("name", Country("country")))
-
-    implicit val testSchemaZonedDateTime: Schema[ZonedDateTime] = Schema.schemaForZonedDateTime.encodedExample(expectedDateTime)
-    implicit val testSchemaBook = implicitly[Schema[Book]].encodedExample(circeCodec[Book].encode(expectedBook))
-
-    val endpoint_with_dateTimes = endpoint.post.in(jsonBody[ZonedDateTime]).out(jsonBody[Book])
-
-    val actualYaml = OpenAPIDocsInterpreter.toOpenAPI(endpoint_with_dateTimes, Info("Examples", "1.0")).toYaml
-    val actualYamlNoIndent = noIndentation(actualYaml)
-
-    actualYamlNoIndent shouldBe expectedYaml
-  }
-
-  test("should include max items in modified string collection schema") {
-    val expectedYaml = load("expected_valid_modified_array_strings.yml")
-
-    implicit val customObjectWithStringsSchema: Schema[ObjectWithStrings] = implicitly[Derived[Schema[ObjectWithStrings]]].value
-      .modify(_.data)(_.validate(Validator.maxSize[String, List](1)))
-
-    val actualYaml = OpenAPIDocsInterpreter
-      .toOpenAPI(
-        endpoint.out(jsonBody[ObjectWithStrings]),
-        Info("Entities", "1.0")
-      )
-      .toYaml
-
-    val actualYamlNoIndent = noIndentation(actualYaml)
-    actualYamlNoIndent shouldBe expectedYaml
-  }
-
-  test("should include max items in modified object collection schema") {
-    val expectedYaml = load("expected_valid_modified_array_objects.yml")
-
-    implicit val customObjectWithStringsSchema: Schema[ObjectWithList] = implicitly[Derived[Schema[ObjectWithList]]].value
-      .modify(_.data)(_.validate(Validator.maxSize[FruitAmount, List](1)))
-
-    val actualYaml = OpenAPIDocsInterpreter
-      .toOpenAPI(
-        endpoint.out(jsonBody[ObjectWithList]),
-        Info("Entities", "1.0")
-      )
-      .toYaml
-
-    val actualYamlNoIndent = noIndentation(actualYaml)
-    actualYamlNoIndent shouldBe expectedYaml
-  }
 }
 
 case class F1(data: List[F1])
```

Comment:
are all of these classes used by the remaining tests?

If so, I'd move them inside an `object VerifyYamlTest` and `import VerifyYamlTest._` here. If some are shared, it's fine to copy them (since these are tests), so that the tests are independent. This is already partially done for coproduct.

Reply:
ok, adjusted